protocols/3308b4/3308b4.ot2.apiv2.py

```diff
@@ -0,0 +1,89 @@
+"""PROTOCOL."""
+metadata = {
+    'protocolName': 'PCR/qPCR Prep',
+    'author': 'Opentrons',
+    'source': 'Custom Protocol Request',
+    'apiLevel': '2.11'   # CHECK IF YOUR API LEVEL HERE IS UP TO DATE
+                         # IN SECTION 5.2 OF THE APIV2 "VERSIONING"
+}
+
+
+def run(ctx):
+
+    [mount_side
```

Comment:
please rename `mount_m300` (bear with me as I create the `fields.json` semantics check to our fields variable names